### ISSUE TITLE
Add x86 release

### ISSUE THREAD

**Issue by patriksvensson** [JanDeDobbeleer/oh-my-posh#298]
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] I looked for duplicate issues before submitting this one

### Description

It would be nice to have a x86 release of oh-my-posh.

The main reason for this is that I would like to use it on my Surface Pro X, which is an ARM64 device. Go is however not supported on ARM64 yet, but it supports x86 emulation, so having an x86 would unblock this scenario.

I'm of course willing to help out with this if you lack the time.

**Comment by JanDeDobbeleer** [comment]
We could do an experimental build to see if this all works according to expectations. Adding that build as output isn't the tricky part, validating it could might be. I'll see what I can do.

@patriksvensson to be clear, we're talking about a Windows x86 build, right?

**Comment by lnu** [comment]
While adding an x86 build we can keep an eye on this: https://github.com/golang/go/issues/36439

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Yes, We're talking about Windows x86 builds.

If it becomes too much work or if it becomes difficult to validate or maintain, we could skip adding x86 builds. It's not super important to me and no one else seemed to have asked for it.

**Comment by JanDeDobbeleer** [comment]
@patriksvensson I'm going to try to give you one for the new year 😉

**Comment by JanDeDobbeleer** [comment]
[posh-windows-386.zip](https://github.com/JanDeDobbeleer/oh-my-posh3/files/5759028/posh-windows-386.zip)

Let me know if this works, because it was waaaay to easy to get this build meaning I'm highly skeptical this is correct. It checks out as a `PE32 executable (console) Intel 80386 (stripped to external PDB), for MS Windows` executable so 🤞🏻

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Awesome! I'm going to try it out later this afternoon or evening when I get some free time 👍

**Comment by JanDeDobbeleer** [comment]
@patriksvensson did you find a moment yet to validate this?

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Sorry! Been a bit busy. Checking it out now!

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Seems to work fine! 

![image](https://user-images.githubusercontent.com/357872/103742965-df7f9c80-4ffb-11eb-85d8-251765d1a765.png)

**Comment by JanDeDobbeleer** [comment]
I'll add that to the build output this week 👌🏻

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Awesome! Looking forward to updating my profile scripts 😀

**Comment by JanDeDobbeleer** [comment]
@patriksvensson took me a few tries, but it should be there. Can you validate the binary in release [3.69.0](https://github.com/JanDeDobbeleer/oh-my-posh3/releases/tag/v3.69.0)?

**Comment by JanDeDobbeleer** [comment]
@patriksvensson the releases contain an x86 build, I'm closing this issue ;-)

**Comment by patriksvensson** [comment]
@JanDeDobbeleer Sorry! I forgot to answer your question. Yes, it's working as expected!
